source/html/renderer/src/card-elements.ts

```diff
@@ -470,7 +470,7 @@ export class FactSet extends CardElement {
             return this.speak + '\n';
```

Comment:
Also, I don't think the default value is implemented yet in the HTML renderer for Input.ChoiceSet